### PR TITLE
Simultaneous Rover Programming

### PR DESCRIPTION
**Pull request by nmarshak1337**
Rovers now work simultaneously, with some of Brett's suggestions incorporated to make the UI actually work with it.

### CODE REVIEW

File: source/view/index.js [hunk 1]
```diff
@@ -18,7 +18,7 @@ var blocklyNodes = {};
 var graphLines = {};
 var loggerNodes = {};
 var currentBlocklyNodeID = undefined;
-var blocklyExecuting = false;
+//var blocklyExecuting = false;
```
Comment (BrettASwift) [inline source/view/index.js]: Remove commented out code.

**Comment by BrettASwift** [comment]
One comment, then :+1:

**Comment by nmarshak1337** [comment]
Now when we stop execution, it stops on the "current" rover, instead of stopping all of them at once. This now depends on https://github.com/virtual-world-framework/vwf/pull/480. 

Please review!

**Comment by AmbientOSX** [comment]
Be sure to do a rebase then :+1:

**Comment by nmarshak1337** [comment]
Discovered a bug after rebase. Please review my fix.

File: source/view/index.js [hunk 2]
```diff
@@ -73,7 +72,7 @@ vwf_view.firedEvent = function( nodeID, eventName, eventArgs ) {
                 break;
 
             case "blocklyStarted":
-                startBlocklyButton.className = "reset";
+                //startBlocklyButton.className = "reset";
```
Comment (BrettASwift) [inline source/view/index.js]: Should this commented code be removed entirely?

**Comment by BrettASwift** [comment]
One comment, but otherwise :+1:

**Comment by BrettASwift** [comment]
Remember to delete branches when you're done with them. GirHub gives the option after a merge.

**Pull request merged**